src/importer/ImportFrame.ts

```diff
@@ -43,10 +43,8 @@ export class ImportFrame extends TypedEventManager<IImportFrameEvents> {
    */
   public close(): void {
     removeClass(document.body, 'flatfile-active')
+    this.$iframe?.remove()
```

Comment:
Why does this exist outside the flatfile-sdk check?

Reply:
flatfile-sdk is our default mounting point, however after adding the mountOn option our iframe could be mounted in a different container.